product-service/serverless.ts

```diff
@@ -0,0 +1,59 @@
+import type { Serverless } from 'serverless/aws';
+
+const serverlessConfiguration: Serverless = {
+  service: {
+    name: 'product-service',
+    // app and org for use with dashboard.serverless.com
+    // app: your-app-name,
+    // org: your-org-name,
+  },
+  frameworkVersion: '2',
+  custom: {
+    webpack: {
+      webpackConfig: './webpack.config.js',
+      includeModules: true
+    }
+  },
+  // Add the serverless-webpack plugin
+  plugins: ['serverless-webpack'],
+  package: { individually: true },
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs12.x',
+    stage: "${opt:stage, 'dev'}",
+    apiGateway: {
+      minimumCompressionSize: 1024,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+    },
+  },
+  functions: {
+    getProductsList: {
+      handler: 'handler.getProductsList',
+      events: [
+        {
+          http: {
+            method: 'get',
+            path: 'products',
+            cors: true
+          }
+        }
+      ]
+    },
+    getProductsById: {
+      handler: 'handler.getProductsById',
+      events: [
+        {
+          http: {
+            method: 'get',
+            path: 'products/{productId}',
+            cors: true
+          }
+        }
+      ]
+    }
+  }
+}
+
+module.exports = serverlessConfiguration;
```

Comment:
Why to mix commonJS and ES modules in one file/project?

Reply:
oh, it's auto generated file, will fix right now. Thanks

Reply:
Didn't find a fast way to switch from commonJS to ES module export for serverless.ts file, serverless throws an error
"service" property is missing in serverless.ts
will fix it in future